courses/machine_learning/deepdive2/feature_engineering/solutions/1_bqml_basic_feat_eng_bqml.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<p>Nice, good job checking that the data already exists! Looks like you have an example of the gsutil command here too. If you haven't come across it yet, this is how you would have students download files from a bucket: <a href="https://cloud.google.com/storage/docs/gsutil/commands/cp" target="_blank">https://cloud.google.com/storage/docs/gsutil/commands/cp</a> (for your other lab, not this one)</p><p> </p><p>Edit: It doesn't look like we use the bucket in this lab. If not, should be safe to delete the bucket creation code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Bucket code removed.